capa/features/extractors/base_extractor.py

```diff
@@ -106,13 +106,14 @@ class StaticFeatureExtractor:
 
     __metaclass__ = abc.ABCMeta
 
-    def __init__(self):
+    def __init__(self, hashes: SampleHashes):
         #
         # note: a subclass should define ctor parameters for its own use.
         #  for example, the Vivisect feature extract might require the vw and/or path.
         # this base class doesn't know what to do with that info, though.
         #
         super().__init__()
+        self.__sample_hashes = hashes
```

Comment:
sorry, hate to churn here, but please use a single underscore. 

double underscores used to have a special meaning in python, having to do with name mangling. they don't anymore, but we stick to one underscore for consistency.